sat/cli/bmccreds/creds_manager.py

```diff
@@ -93,10 +107,18 @@ def _get_bmc_type(bmc):
         return 'Unknown'
 
     @staticmethod
-    def _generate_random_password_string():
-        """Get a random string of letters and numbers that is RANDOM_PASSWORD_LENGTH characters long."""
+    def _generate_random_password_string(length, allowed_chars):
```

Comment:
minor: you could change this method to no longer be a static method, and then it could reference the `length` and `allowed_chars` directly from the instance attributes. E.g. `self.length` and `self.allowed_chars`.

Reply:
Addressed